security-demo/release/log/README.md

```diff
@@ -0,0 +1 @@
+# Log files place holder
```

Comment:
Is this file necessary?

Reply:
Removed , handled in build-release.sh script.